cpu/fe310/periph/spi.c

```diff
@@ -29,9 +29,14 @@
 
 #include "vendor/spi.h"
 
-#define ENABLE_DEBUG        (0)
+#define ENABLE_DEBUG    (0)
 #include "debug.h"
 
+#define SPI_CLK_NUMOF   ARRAY_SIZE(_spi_clks)
+
+/* DIV_UP is division which rounds up instead of down */
+#define SPI_DIV_UP(a,b) (((a) + ((b) - 1)) / (b))
```

Comment:
```suggestion
#define SPI_DIV_UP(a, b) (((a) + ((b) - 1)) / (b))
```
When touching this anyway, maybe you could sneak in this as well?

Reply:
Please squash right away. No need to let Murdock run twice.

Reply:
Fixed :-)